### PR TITLE
adding note

### PR DESCRIPTION
**Pull request by Steinsplitter**
adding note regarding copyrights

### CODE REVIEW

**Comment by misaochan** [comment]
Looks good to me, but will wait for @tobias47n9e (he maintains the website).

**Comment by nicolas-raoul** [comment]
The bullet list is about what is on-topic and what is off-topic.
About copyright I think we should add a separate note, as it is transversal.

Let's imagine I want **only cats and hamsters**, that are **vaccinated**. If I post these requirements:
```
✔ Cats
✔ Hamsters
✔ Vaccinated animals
✗ Canaries
✗ Tortoises
✗ Non-vaccinated animals
```
... then it sounds like vaccinated dogs are OK. In our case, it sounds like all photos taken by oneself are OK, which is not what we want to convey.
Instead, a better requirements list is:
```
✔ Cats
✔ Dogs
✗ Canaries
✗ Tortoises
NOTE: Only vaccinated animals
```

**Comment by misaochan** [comment]
@nicolas-raoul Good point. We could go with that.

**Comment by Steinsplitter** [comment]
Modified inf5f0527512e22adc340be57512420e3c0fd9948e

File: index.html [hunk 1]
```diff
@@ -147,10 +147,24 @@ <h2>Photo Guidelines</h2>
                 The following guidelines will help you to understand what photos the community is looking for. There is flexibility in these guidelines and they don't cover all cases. If you are unsure, just upload your photo and discuss it with the community.
             </p>
             <ul class="guide-list">
-                <li class="text-success">✓ Photos that document the world around you</li>
                 <li class="text-success">✓ Photos of notable objects that you find in the <strong>Nearby List</strong> in the app.</li>
+                <li class="text-success">✓ Photos that you created entirely yourself.</li>
```
Comment (nicolas-raoul) [inline index.html]: I believe this line is not necessary, right?

Comment (Steinsplitter) [reply index.html]: 🆗

File: index.html [hunk 2]
```diff
@@ -147,10 +147,23 @@ <h2>Photo Guidelines</h2>
                 The following guidelines will help you to understand what photos the community is looking for. There is flexibility in these guidelines and they don't cover all cases. If you are unsure, just upload your photo and discuss it with the community.
             </p>
             <ul class="guide-list">
-                <li class="text-success">✓ Photos that document the world around you</li>
                 <li class="text-success">✓ Photos of notable objects that you find in the <strong>Nearby List</strong> in the app.</li>
                 <li class="text-danger">✖ Photos of you or your friends. But if you are documenting an event it doesn't matter if they are in the picture.</li>
-                <li class="text-danger">✖ Photos of poor quality. Make sure the things you are trying to document are visible on the picture.</li>
+                <li class="text-danger">✖ Photos of poor quality. Make sure the things you are trying to document are visible on the picture.</li>            </ul>
```
Comment (nicolas-raoul) [inline index.html]: It is always a good idea to check https://github.com/commons-app/commons-app.github.io/pull/27/files to see whether your PR changes exactly what you want. For instance, the line above should not have been affected by your PR, which tips that the indentation is not perfect.

Comment (Steinsplitter) [reply index.html]: Please make the necessary modifications yourself. You may consider moving the help page onwiki so that the community can edit it.

**Comment by sivaraam** [comment]
Hey @Steinsplitter,

I tried to see what you have added in this change, but couldn't find it. Could you please let me know the part of the diff that corresponds particularly to what you are doing (it seems messed up now)?

It would be great if you could, close this and open a new PR with just the changes you wanted to make.

Thanks.

**Pull request closed without merging**